tools/odbc/statement_functions.cpp

```diff
@@ -818,6 +818,7 @@ SQLRETURN duckdb::ExecuteStmt(SQLHSTMT statement_handle) {
 	return duckdb::BatchExecuteStmt(hstmt);
 }
 
+// https://learn.microsoft.com/en-us/sql/odbc/reference/syntax/sqlbindparameter-function?view=sql-server-ver16
```

Comment:
This isn't the `SQLBindParameter` function, could you add this link to that function instead?

Reply:
It's not...it's a DuckDB specific method, but the method signature matches so I thought the link is helpful. If you look at the SetDiagnosticRecord() messages all three have `SQLBindParameter`. Did some renaming happen at some point? 

What's more important is the [TODO](https://github.com/duckdb/duckdb/blob/f444a73249829caf5184324a178b3c2d81bb8131/tools/odbc/statement_functions.cpp#L856) listed further down...